File: Modul 4/Bahasa Python/PRAK404-2310817120001-DinaIzzatiElfadheya.py
while True:
  print("\nPilih program")
  print("1. Penjumlahan")
  print("2. Pengurangan")
  print("3. Perkalian")
  print("4. Pembagian")
  print("5. Exit")

  pilihan = int(input("Masukkan Pilihan : "))

  if pilihan == 5:
    print("Terimakasih, telah menggunakan kalkulator DINA_IZZATI_ELFADHEYA")
    break
  
  if pilihan < 1 or pilihan > 5:
    print("Input anda salah, silahkan coba lagi")
    continue

  a = int(input("Masukkan nilai pertama : "))
  b = int(input("Masukkan nilai kedua : "))

  if pilihan == 1:
    hasil = a + b
  elif pilihan == 2:
    hasil = a - b
  elif pilihan == 3:
    hasil = a * b
  else:
    hasil = a / b

  print(f"Hasil perkalian antara {a:.2f} dan {b:.2f} adalah {hasil:.2f}")

File: Modul 4/Bahasa C/PRAK404-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>
int main() {
  
  int pilihan;
  float nilai_pertama, nilai_kedua, hasil;
  
  while(1) {

    printf("\nPilih program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");

    printf("Masukkan Pilihan : ");
    scanf("%d", &pilihan);

    if(pilihan == 5) {
      printf("Terimakasih, telah menggunakan kalkulator DINA_IZZATI_ELFADHEYA\n");
      break;
    }

    if(pilihan < 1 || pilihan > 5) {
      printf("Input anda salah, silakan coba lagi\n");
      continue;
    }

    printf("Masukkan nilai pertama : ");
    scanf("%f", &nilai_pertama);

    printf("Masukkan nilai kedua : ");  
    scanf("%f", &nilai_kedua);

    if(pilihan == 1) {
      hasil = nilai_pertama + nilai_kedua;
    } else if(pilihan == 2) {
      hasil = nilai_pertama - nilai_kedua;
    } else if(pilihan == 3) {
      hasil = nilai_pertama * nilai_kedua; 
    } else {
      hasil = nilai_pertama / nilai_kedua;
    }

    printf("Hasil perkalian antara %.2f dan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
  }
}
